docs/README.md

```diff
@@ -20,8 +20,10 @@ installed. Also install the following libraries:
     $ sudo pip install Pygments
     # Following is needed only for generating API docs
     $ sudo pip install sphinx
-    $ Rscript -e 'install.packages(c("knitr", "devtools"), repos="http://cran.stat.ucla.edu/")'
+    $ sudo Rscript -e 'install.packages(c("knitr", "devtools", "roxygen2", "testthat"), repos="http://cran.stat.ucla.edu/")'
```

Comment:
So it definitely didn't work on my machine - I think it depends on how your R is set up (for example on my machine lib points to `/usr/local/lib/R/site-library`). I _think_ having sudo would make it consistent (since I think users can also setup pip/gem to default to user specific or site wide).